react/Empty/index.jsx

```diff
@@ -19,8 +19,14 @@ export const Empty = ({
   ...restProps
 }) => {
   if (layout) {
-    depreciationLogger(`Please add layout=false in the props of the Empty component, and implement the vertical
-    centering with a custom layout component. The use of Empty without layout=false is deprecated.`)
+    depreciationLogger(
+      `"layout" prop on Empty component is deprecated. Please add a style to the parent component for manage the position of the empty component and remain compatible with future major versions of cozy-ui`
+    )
```

Comment:
⛏️ ♻️ on peut peut-être supprimer le `and remain compatible with future major versions of cozy-ui` ?